src/transforms/wasm.rs

```diff
@@ -94,19 +95,31 @@ impl Wasm {
     }
 }
 
-impl Transform for Wasm {
-    fn transform(&mut self, output: &mut Vec<Event>, event: Event) {
-        self.module
-            .process(event)
-            .map(|outputs| outputs.into_iter().next())
-            .unwrap_or(None)
+impl TaskTransform for Wasm {
+    fn transform(
+        self: Box<Self>,
+        task: Box<dyn Stream01<Item = Event, Error = ()> + Send>,
+    ) -> Box<dyn Stream01<Item = Event, Error = ()> + Send> {
+        let mut inner = self;
+
+        Box::new(
+            task.filter_map(move |event| {
+                inner
+                    .module
+                    .process(event)
+                    .map(|events| futures01::stream::iter_ok(events))
+                    .ok()
```

Comment:
I felt it was a bit weird to just drop any errors, but this maintains the current behavior.

Reply:
WASM events are able to output events to stderr but later this will need to change to support reporting errors up to our internal metrics.